File: Python/Program/Quest.py
class Quest:
    def __init__(self, name: str, description: str, reward: list, EXP: int):
        self._name = name
        self._description = description
        self._reward = reward
        self._EXP = EXP

    def get_name(self) -> str:
        return self._name

    def set_name(self, name: str):
        self._name = name

    def get_description(self) -> str:
        return self._description

    def set_description(self, description: str):
        self._description = description

    def get_reward(self) -> list:
        return self._reward

    def set_reward(self, reward: list):
        self._reward = reward

    def get_EXP(self) -> int:
        return self._EXP

    def set_EXP(self, EXP: int):
        self._EXP = EXP

File: C++/Program/Quest.cpp
#pragma once
#include <iostream>
#include <string>
#include <vector>

using namespace std;

class Quest {
private:
    string name;
    string description;
    vector<string> reward;
    int EXP;

public:
    // Constructor
    Quest(){}

    Quest(string name, string description, vector<string> reward, int EXP) {
        this->name = name;
        this->description = description;
        this->reward = reward;
        this->EXP = EXP;
    }

    // Getter and Setter for name
    string getName() {
        return name;
    }

    void setName(string name) {
        this->name = name;
    }

    // Getter and Setter for description
    string getDescription() {
        return description;
    }

    void setDescription(string description) {
        this->description = description;
    }

    // Getter and Setter for reward
    vector<string> getReward() {
        return reward;
    }

    void setReward(vector<string> reward) {
        this->reward = reward;
    }

    // Getter and Setter for EXP
    int getEXP() {
        return EXP;
    }

    void setEXP(int EXP) {
        this->EXP = EXP;
    }
};